poetry/repositories/installed_repository.py

```diff
@@ -49,20 +49,19 @@ def load(cls, env):  # type: (Env) -> InstalledRepository
                     is_standard_package = False
 
                 if is_standard_package:
-                    if (
-                        path.name.endswith(".dist-info")
-                        and env.site_packages.joinpath(
-                            "{}.pth".format(package.pretty_name)
-                        ).exists()
-                    ):
-                        with env.site_packages.joinpath(
-                            "{}.pth".format(package.pretty_name)
-                        ).open() as f:
+                    pth_file = env.site_packages.joinpath(
+                        "{}.pth".format(package.pretty_name)
+                    )
+                    if path.name.endswith(".dist-info") and pth_file.exists():
+                        with pth_file.open() as f:
                             directory = Path(f.readline().strip())
```

Comment:
```suggestion
                    directories = set()
                    site.addpackage(env.site_packages.as_posix(), "{}.pth".format("package.pretty_name"), directories)
                    if directories:
                        # TODO: handle multiple source directories
                        package.source_type = "directory"
                        package.source_url = directories.pop()
                        continue
```

Reply:
Where does `site.addpackage` come from? I don't see the string `addpackage` anywhere in the code base.

Reply:
```
>>> import site
>>> site.addpackage
<function addpackage at 0x7f9a39dfd310>
```